src/lib/page-action.js

```diff
@@ -10,6 +10,12 @@ export default class PageAction {
     browser.webNavigation.onCompleted.addListener(({ frameId, tabId }) => {
       this.create(frameId, tabId);
     });
+    browser.webNavigation.onHistoryStateUpdated.addListener((
+      { frameId, tabId }
+    ) =>
```

Comment:
Debatably yes, but I'm using [prettier](https://prettier.github.io/prettier/) for style, and this is what it does 🤷‍♂️